tests/syscalls/syscalls_test.go

```diff
@@ -54,6 +57,73 @@ var _ = Describe("Syscalls", func() {
 		KspDeleteAll()
 	})
 
+	Describe("Annotation", func() {
+		It("can annotate pre existing pod", func() {
+			k8sClient := GetK8sClient()
+			// ReplicaSet
+			replicasetSelector := metav1.ListOptions{
+				LabelSelector: "app=nginx-replicaset",
+			}
+			rspods, err := k8sClient.K8sClientset.CoreV1().Pods("nginx").List(context.TODO(), replicasetSelector)
+			Expect(err).To(BeNil())
+			rspodName := rspods.Items[0].Name
+
+			rsannotated := false
+			for key, value := range rspods.Items[0].Annotations {
+				log.Printf("K8sGetPods pod=%s ns=%s Key=%v value=%s", rspods.Items[0].Name, "nginx", key, value)
+				if key == "kubearmor-policy" {
+					rsannotated = true
+				}
+
+			}
```

Comment:
Can we use the existing helper functions for getting and matching pods here? :thinking: